src/app/views/river-detail/components/main-tab/components/beta-box/beta-box.vue

```diff
@@ -193,18 +139,28 @@ export default {
   },
   data: () => ({
     editModalVisible: false,
-    formData: {}
+    formData: {},
+    reachDetail: null,
   }),
   computed: {
     ...mapState({
       loading: state => state.RiverDetail.loading,
       river: state => state.RiverDetail.data,
       editMode: state => state.Global.editMode,
-      gages: state => state.RiverGages.data?.gauges ?? [],
-      metrics: state => state.RiverGages.data?.metrics ?? []
     }),
-    readingIsEstimated () {
-      return this.river?.readingsummary?.gauge_estimated
+    // TODO: decide how to handle secondary gauges. do we fall back if primary isn't reporting?
```

Comment:
cc @AWKevin - I think we decided that not falling back is ultimately more clear.